Libraries/StyleSheet/processTransform.js

```diff
@@ -25,7 +25,7 @@ var stringifySafe = require('stringifySafe');
  * be applied in an arbitrary order, and yet have a universal, singular
  * interface to native code.
  */
-function processTransform(transform: Object): Object {
+function processTransform(transform: Array<Object>): Array<Object> | Array<number> {
```

Comment:
Does this change the public API? Now people have to pass:

`[{scale: 2}, {translateX: 10}]`

Is it the same API as before?

And why not pass:

`{scale: 2, translateX: 10}`

I guess it's because the array defines the order of transforms?